### DISCUSSION TITLE
Mobile App - Regarding the changelog of 1.7.2 beta

### DISCUSSION THREAD

**Discussion by darkred** [community/community#4057] [Mobile] [answered]
It's about beta version 1.7.2 of GitHub mobile app (in Android 10).  It was out two months ago.

In its changelog ([link](https://webcache.googleusercontent.com/search?q=cache:T1H9jzICp5QJ:https://www.apkmirror.com/apk/github/github-2/github-2-1-7-2-release/github-1-7-2-android-apk-download/+&cd=1&hl=en&ct=clnk)|WHAT'S NEW tab) it said that:
> New: Now it is possible to get an overview of unread activity in the issues and pull requests list views.
<details>
<summary>Play Store screenshot:</summary>

![](https://user-images.githubusercontent.com/723651/120078564-ba79ec00-c0b8-11eb-917b-4cf80d224ff5.jpg)
</details>

Could you please explain in what part of the app does it refer to? 
In Home|My Work|Issues ? In the 'Notifications' tab? In "Issues" list of repos??
I can't spot a difference anywhere.

P.S. I had contacted `mobilefeedback+android@github.com` at that time but never got a response.

**Comment by aconsuegra** [comment, marked as answer]
Hi @darkred, sorry about the late reply. This new feature applies to `Home|My Work|Issues` and `Home|My Work| Pull Requests`. It simply displays a blue dot in each row with unread activity, like it happens in Notifications.

**Reply by darkred** [reply]
Thanks a lot for your response!
